lib/util/call-taker.js

```diff
@@ -1,9 +1,66 @@
+import {isTransit} from '@opentripplanner/core-utils/lib/itinerary'
+import {randId} from '@opentripplanner/core-utils/lib/storage'
+
 import {getRoutingParams} from '../actions/api'
+import {getTimestamp} from './state'
+
+export const TICKET_TYPES = {
+  own_tickets: 'Will use own tickets',
+  hop_new: 'Will purchase new Hop Card',
+  hop_reload: 'Will reload existing Hop Card'
+}
+
+const PAYMENT_PREFS = {
+  request_call: 'Call requested at provided phone number',
+  phone_cc: 'Will call in credit card info to TriMet',
+  fax_cc: 'Will fax credit card info to TriMet',
+  mail_check: 'Will mail check to TriMet'
+}
+
+const positiveIntInputProps = {
+  min: 0,
+  step: 1,
+  type: 'number'
+}
+
+export const GROUP_FIELDS = [
+  {inputProps: positiveIntInputProps, fieldName: 'numStudents', label: 'students 7 or older'},
+  {inputProps: positiveIntInputProps, fieldName: 'numFreeStudents', label: 'students under 7'},
+  {inputProps: positiveIntInputProps, fieldName: 'numChaperones', label: 'chaperones'}
```

Comment:
Age and/or label should be configurable.

Reply:
Again, I think that's a bridge we should cross if/when this module ever becomes generalized beyond TriMet.

Reply:
Make an issue or point to some kind of roadmap item so this doesn't get lost. Add a comment in the code that technical debt is being added with a plan to pay it off in the future.

Reply:
Issue is here: https://github.com/opentripplanner/otp-react-redux/issues/353